src/templates/comment-reply-input.jsx

```diff
@@ -145,12 +145,6 @@ const CommentReplyInput = React.createClass({
                 rowCount: 1,
             });
         }
-
-        // Force scroll the parent window to the top on iPhone
-        // Note: This can be removed if we move away from using an iFrame for the notifications window
-        if (window.parent && hasTouch()) {
```

Comment:
@dmsnell what do you think about adding a check here to see if this is loaded in an `iFrame`, and conditionally perform the `scrollTo` based on that?